web/app.js

```diff
@@ -2152,15 +2172,15 @@ function webViewerInitialized() {
 
   if (!PDFViewerApplication.supportsPrinting) {
     appConfig.toolbar.print.classList.add("hidden");
```

Comment:
Don't you need to guard against an undefined toolbar here?
```suggestion
    appConfig.toolbar?.print.classList.add("hidden");
```